File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/EntryPrice.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EntryPrice {

    @SerializedName("price_raw")
    @Expose
    private int priceRaw;
    @SerializedName("price")
    @Expose
    private String price;

    public int getPriceRaw() {
        return priceRaw;
    }

    public void setPriceRaw(int priceRaw) {
        this.priceRaw = priceRaw;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itemlist/EntryPrice.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EntryPrice {

    @SerializedName("price_raw")
    @Expose
    var priceRaw: String? = null

    @SerializedName("price")
    @Expose
    var price: String? = null

}
